Cython/Utility/ObjectHandling.c

```diff
@@ -1037,6 +1037,19 @@ static CYTHON_INLINE int __Pyx_PyDict_ContainsTF(PyObject* item, PyObject* dict,
 
 /////////////// PySetContains.proto ///////////////
 
+static int __Pyx_PySet_ContainsUnhashable(PyObject *set, PyObject *key); /* proto */
+
+static CYTHON_INLINE int __Pyx_PySet_ContainsTF(PyObject* key, PyObject* set, int eq) {
```

Comment:
Almost. ;)
I really just meant that this one signature should go into the `.proto` section. All the rest can go into the implementation, also the `...Unhashable` function. Since the latter is not called by user generated code but only by the global `...ContainsTF` helper function that *follows* it, there is no need to pre-declare it.

Reply:
Ok, I had miunserstood. This should now be fixed.